roles/keystone/templates/etc/keystone/keystone-paste.ini

```diff
@@ -81,13 +84,13 @@ paste.app_factory = keystone.service:v3_app_factory
 paste.app_factory = keystone.service:admin_app_factory
 
 [pipeline:public_api]
-pipeline = sizelimit url_normalize build_auth_context token_auth json_body ec2_extension user_crud_extension{{ ' passval unauthlog' if keystone.jellyroll else '' }} public_service
+pipeline = sizelimit url_normalize build_auth_context token_auth json_body ec2_extension user_crud_extension{{ 'rbac_filter passval unauthlog' if keystone.jellyroll else '' }} public_service
```

Comment:
There is a space after the opening quote on purpose, please put it back, in all three of these.